ports/libsimpleservo/capi/Cargo.toml

```diff
@@ -31,6 +31,8 @@ canvas2d-azure = ["simpleservo/canvas2d-azure"]
 canvas2d-raqote = ["simpleservo/canvas2d-raqote"]
 debugmozjs = ["simpleservo/debugmozjs"]
 default = ["webdriver", "max_log_level"]
+media-dummy = ["libservo/servo-media-dummy"]
+media-gstreamer = ["libservo/servo-media-gstreamer"]
```

Comment:
Both of these crates depends on simpleservo, not libservo.

Reply:
It's possible I'm confused by the project structure. `components/servo/Cargo.toml` defines the two features `servo-media-dummy` and `servo-media-gstreamer`, which are selected implicitly when enabling libsimpleservo's `media-dummy` or `media-gstreamer` feature. Same for the glutin port. And depending on which feature is selected, one of the `media_platform` modules is built conditionally in libservo.

What am I missing (that would help resolve the issue)?

Reply:
`capi` and `jniapi` depend on the common `libsimpleservo/api` crate, and that one has the expected `libservo` dependency. Hmm...

Reply:
I'll add `media-dummy` and `media-gstreamer` features to `libsimpleservo/api`, to be used by `capi` and `jniapi`, since `api` is the common crate which depends on `libservo`.